### PR TITLE
chore(backup): update take backup response code

### PR DESCRIPTION
**Pull request by PHWaechtler**
related to OPT-6568

## What is the purpose of the change

Update the response code for the take backup API as adjusted with OPT-6568 (https://github.com/camunda/camunda-optimize/pull/5466)

## Are there related marketing activities

No

## When should this change go live?

With the next release

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/self-managed/backup-restore/optimize-backup.md [hunk 1]
```diff
@@ -44,12 +44,12 @@ POST actuator/backup
...
-| 409 Conflict     | Indicates that a backup with the same `backupId` already exists.                                                                                                                                                                                              |
-| 500 Server Error | All other errors, e.g. issues communicating with Elasticsearch for snapshot creation. Refer to the returned error message for more details.                                                                                                                   |
+| Code             | Description                                                                                                                                 |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| 202 Accepted     | Backup process was successfully initiated. To determine whether backup process has completed refer to the GET API.                          |
```
Comment (andromaqui) [inline docs/self-managed/backup-restore/optimize-backup.md:49]: `To determine whether backup process was completed refer to the GET API.`

wdyt?

**Review by andromaqui — DISMISSED** [review]
LGTM, just one small suggestion :)

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
@PHWaechtler Did you want to make this change in only the `next` version of docs or also `8.1` and `8.0`?

**Comment by christinaausley** [comment]
@PHWaechtler Apologies -- I meant `3.9.0` here. If this is just for `next`, feel free to merge when ready 👍

**Pull request merged**